src/gt4py/storage/storage.py

```diff
@@ -427,6 +437,18 @@ def _construct(cls, backend, dtype, default_origin, shape, alignment, layout_map
     def data(self):
         return self._device_field
 
+    def to_numpy(self, copy=False):
+        self.synchronize()
+        if copy:
+            return copy.deepcopy(np.asarray(self))
```

Comment:
```suggestion
            return np.array(self, subok=False)
```
Is the numpy way of doing copies. Maybe needs double checking if this doesn't result in a storage with our numpy patch-hack.